### ISSUE TITLE
The boundary=national_park tag is not rendered when tagged on a closed way (not a relation)

### ISSUE THREAD

**Issue by talllguy** [gravitystorm/openstreetmap-carto#1141]
Hi. I have been working on the [Maryland State Park Import](http://wiki.openstreetmap.org/wiki/Maryland_State_Parks) lately and have been doing a lot of refreshing of tiles to check how things are looking. One thing I've noticed is certain parks that are rendering on Mapbox tiles but not the standard map. **My theory is that ways with `boundary=national_park` are not rendered, while multipolygons with the tag are.**

Here's a (inaccurate) boundary polygon that I'll probably replace. According to [one of the tiles](http://c.tile.openstreetmap.org/13/2333/3111.png/status), it is up to date. The park was drawn months ago and then updated a few days ago, but there is no rendering despite repeated refreshing and `/dirty`ing.
[![screen shot 2014-11-29 at 8 58 49 pm](https://cloud.githubusercontent.com/assets/1827881/5236538/a13d852e-780a-11e4-9889-c236f1e90050.png)](http://www.openstreetmap.org/way/294606587#map=13/39.6489/-77.4418)
[![screen shot 2014-11-29 at 8 58 58 pm](https://cloud.githubusercontent.com/assets/1827881/5236539/a3d75a3a-780a-11e4-99ca-5a734e54f815.png)](http://www.openstreetmap.org/#map=13/39.6496/-77.4358)

Now, I noticed this issue in another place, over at Assateague Island National Seashore. Before I started fiddling with it, that national park was a single closed polygon. As part of the import, I added the Assateague State Park, and made in an inner role of a multipolygon with the old Assateague Island NS boundary as outer. After a couple minutes it shows up at many zoom levels. (The original boundary was brought in around 4 years ago). Here is a map to that [area](http://www.openstreetmap.org/#map=13/38.2277/-75.1286) and the [relation](http://www.openstreetmap.org/relation/4234479#map=10/38.0589/-75.2248).
![screen shot 2014-11-29 at 9 13 47 pm](https://cloud.githubusercontent.com/assets/1827881/5236553/ab14d898-780c-11e4-854d-0012dbe04c1a.png)

So, for Catoctin NP, I plan to replace the boundaries and also stick Camp David into a multipolygon with it, since the park doesn't really include the President's bunker. I bet that will make it show up, but I wanted to raise the issue here before doing so.

I just thought of one more area where a single closed way with `boundary=national_park` does not render. It is [an island / reef in Mexico I worked on a few weeks ago](http://www.openstreetmap.org/way/310123659#map=11/22.4739/-89.7892).

**Comment by aceman444** [comment]
Hi, so can this be solved in any way. The problem still exists today. Sorry for filing a dupe.

**Comment by talllguy** [comment]
@aceman444 you can use [this workaround (above)](https://github.com/gravitystorm/openstreetmap-carto/issues/1141#issuecomment-64991863) until it is fixed.

**Comment by jeisenbe** [comment]
I think that this problem could be solved at the next database reload by adding Can this issue be solved at the next database reload by adding this to `local polygon_values` in openstreetmap-carto.lua
```
    boundary = {national_park = true, protected_area= true}
```
It worked in my test rending, after reloading the database, with no additional changes. 

I could add this to the list for the next database reload, or perhaps we could implement it now, since this feature is already rendered?

**Comment by jeisenbe** [comment]
Fixed by PR #3785 and subsequent database reload.

**Comment by talllguy** [comment]
Coincidentally, I just changed the boundary that triggered this discussion to `boundary=protected_area` due the [planned migration of US Public Lands](https://wiki.openstreetmap.org/wiki/United_States/Public_lands) tagged as `boundary=national_park` to `boundary=protected_area`.

Here's the relation now: https://www.openstreetmap.org/relation/4234479
